src/range.rs

```diff
@@ -288,11 +284,10 @@ impl<V: Version> fmt::Display for Range<V> {
     }
 }
 
-fn interval_to_string<V: Version>(interval: &Interval<V>) -> String {
-    match interval {
-        (start, Some(end)) => format!("[ {}, {} [", start, end),
-        (start, None) => format!("[ {}, ∞ [", start),
-    }
+fn interval_to_string<V: Version>((start, end): &Interval<V>) -> String {
```

Comment:
The destructuring of `(start, end)` is nice but for the body of the function, I'd prefer if we keep a match. We can match on just `end` though.

Reply:
Done ✅